containers-tests/tests/intmap-properties.hs

```diff
@@ -1187,3 +1268,43 @@ prop_fromSet :: [(Int, Int)] -> Bool
 prop_fromSet ys =
   let xs = List.nubBy ((==) `on` fst) ys
   in fromSet (\k -> fromJust $ List.lookup k xs) (IntSet.fromList $ List.map fst xs) == fromList xs
+
+newtype Identity a = Identity a
+
+instance Functor Identity where
+  fmap f (Identity a) = Identity (f a)
+
+instance Applicative Identity where
+  pure a = Identity a
+  Identity f <*> Identity a = Identity (f a)
+
+prop_traverseWithKey :: IntMap () -> Bool
+prop_traverseWithKey mp = mp == newMap
+  where Identity newMap = traverseWithKey (\_ -> Identity) mp
+
+prop_traverseMaybeWithKey :: IntMap () -> Bool
+prop_traverseMaybeWithKey mp = mp == newMap
+  where Identity newMap = traverseMaybeWithKey (\_ -> Identity . Just) mp
+
+#if MIN_VERSION_base(4,8,0)
+test_minimum :: Assertion
+test_minimum = do
+    getOW (minimum testOrdMap) @?= "min"
+    minimum (elems testOrdMap) @?= minimum testOrdMap
+  where getOW (OrdWith s _) = s
```

Comment:
Why are these tests mixed in with the properties? Do we do that elsewhere?

Reply:
I just added new tests at the bottom, I'll put the new unit tests up near the other unit tests.